### PR TITLE
Lowers down and disables some events

### PR DESCRIPTION
**Pull request by Azarak**
## About The Pull Request
on the request of Xyel

## Changelog
:cl:
tweak: Appendicitis is now less likely to occur
tweak: Spacevines are now less likely to occur
tweak: Brand Intelligence event now cant occur
tweak: Spider Infestation is now less likely to occur
tweak: Random heart attack is now less likely to occur
/:cl:

### CODE REVIEW

File: code/modules/events/blob.dm [hunk 1]
```diff
@@ -1,7 +1,8 @@
 /datum/round_event_control/blob
 	name = "Blob"
 	typepath = /datum/round_event/ghost_role/blob
-	weight = 10
+	//weight = 10 //ORIGINAL
+	weight = 5 //SKYRAT EDIT CHANGE
```
Comment (NotRanged) [inline code/modules/events/blob.dm]: Blob is already so rare, I barely see it once a day :(

Comment (EvilCrimsonWitch) [reply code/modules/events/blob.dm]: Aw, I like blob. That's a shame.

Comment (Azarak) [reply code/modules/events/blob.dm]: I dont mind having it be either, convince Xyel as he wants to have it changed

Comment (Azarak) [reply code/modules/events/blob.dm]: No longer reduced

**Comment by SpaceVampire** [comment]
Why is Brand Intelligence being removed? I never heard anyone complain about it. I just wish we could see the reason for these charges in general. I understand Spider Infestation, because we actually had a vote on that one.

**Comment by Azarak** [comment]
> 
> 
> Why is Brand Intelligence being removed? I never heard anyone complain about it. I just wish we could see the reason for these charges in general. I understand Spider Infestation, because we actually had a vote on that one.

@hugmeorelse

**Comment by YakumoChen** [comment]
> Why is Brand Intelligence being removed? I never heard anyone complain about it. I just wish we could see the reason for these charges in general. I understand Spider Infestation, because we actually had a vote on that one.

It removes every vending machine onboard, severely reducing access to station resources, and makes a ton of holes in the hull, and then the vending machines tend to kill people on top of it. A successful brand intelligence is usually a guaranteed shuttle call.

**Comment by Oscari-pog** [comment]
Pointless to decrease appendicitis and random heart attack, happens once per shift and the person is usually treated as soon as they notice something wrong.

Spacevines are usually killed by a assistant with a welder - no reason to decrease them. The only reason for this I can think of is that one round where ultravines were bussed in.

Didnt we already decrease spiders? Spiders are awful at their current state so I cant complain about that one.

**Review by KathrinBailey — APPROVED** [review]

**Comment by hugmeorelse** [comment]
> Pointless to decrease appendicitis and random heart attack, happens once per shift and the person is usually treated as soon as they notice something wrong.
> 
> Spacevines are usually killed by a assistant with a welder - no reason to decrease them. The only reason for this I can think of is that one round where ultravines were bussed in.
> 
> Didnt we already decrease spiders? Spiders are awful at their current state so I cant complain about that one.

There is an opportunity cost with each event. Each event type that occurs blocks possibly more interesting events from happening. There's an upper bound that limits the number of events that can possibly happen per round, and it cannot go any higher unless the round gets extended. Would you prefer something more interesting to happen or appendicitis or heart attacks that get fixed right away? You can have multiples of them per round, and I have seen it happen quite often. Furthermore, events like these are highly disruptive to an individual and can hardly be called interesting or fun for them. They may produce some entertainment for a doctor, but that's about it if they can call it that. Either ways, only a grand total of two people are affected. It's not really much of an event.

As for spacevine, I never see them do anything of note. It's just there which is perfectly fine, but they can happen multiple times per round as well. This is fine, but it's also dull. People don't want anything as crazy as spiders, but they also think the station is too chill at times too.

Secondly, weights are all relative. Lowering the weights of one event increases the weights of others. Removing one event does the same thing. Some of these are adjustments compensating for that or to to increase the weights of other events without having to touch them.

Overall, balancing events is difficult. The only way we can gauge them is by making changes and seeing how it impacts the rhythm of a round.

**Comment by hugmeorelse** [comment]
> Why is Brand Intelligence being removed? I never heard anyone complain about it. I just wish we could see the reason for these charges in general. I understand Spider Infestation, because we actually had a vote on that one.



> > Why is Brand Intelligence being removed? I never heard anyone complain about it. I just wish we could see the reason for these charges in general. I understand Spider Infestation, because we actually had a vote on that one.
> 
> It removes every vending machine onboard, severely reducing access to station resources, and makes a ton of holes in the hull, and then the vending machines tend to kill people on top of it. A successful brand intelligence is usually a guaranteed shuttle call.

Yakumo hit the nail on the head there. You can also be in mid-RP discussion near the vending machine when it suddenly blows up and maims both you and your friends. These were disabled in the oldbase for a reason. Furthermore, I want to start clamping down on items that take away from the atmosphere of the server. We're already looking to remove meat sheets, pizza sheets, and other items that are overly silly. These things will take time, but we need to start somewhere.

**Pull request merged**